java/test/apps/gui3/FirstTimeStartUpWizardTest.java

```diff
@@ -58,7 +58,8 @@ public void createAndDisplayFrame() {
                 // called when wizard is disposed, but do nothing in tests
             }
         };
-        FirstTimeStartUpWizard t = new FirstTimeStartUpWizard(new jmri.util.JmriJFrame("Decoder Pro Wizard", false, false), a);
+        jmri.util.JmriJFrame jf = new jmri.util.JmriJFrame("Decoder Pro Wizard", false, false);
+        FirstTimeStartUpWizard t = new FirstTimeStartUpWizard(jf, a);
         Assert.assertNotNull("exists", t);
         t.dispose();
         SwingTestCase.disposeFrame("Decoder Pro Wizard", false, false);
```

Comment:
If you have a reference to the frame already, you can simply call dispose directly on it (use ```jf.dispose()``` in this case instead of the SwingTestCase method.